middleware/oauth.global.ts

```diff
@@ -21,7 +25,34 @@ export default defineNuxtRouteMiddleware(async (to) => {
       }
     }
   }
+
+  // Check authentication
   if (authStrategy === "auth0" && !loggedIn.value && to.path !== "/login") {
     return router.push("/login");
   }
+
+  // Check role-based access for /config route
+  if (to.path === "/config" && loggedIn.value && user.value && !ci) {
```

Comment:
```suggestion
  if (authStrategy === auth0 && loggedIn.value && user.value && to.path === "/config") {
```

I think the RBAC check was implicitly skipped if `authStrategy === "none"` because `user.value` would be null, but let's be explicit and use the `authStrategy` variable since we set it in env, similar to what we do for the login page.

Also, instead of adding on `!ci` here, and we just do this at the very beginning?

```javascript
  const ci = process.env.CI;
  
  // Bypass all middleware checks in CI environment
  if (ci) {
    return;
  }
```

Maybe we can do something similar eventually if `authStrategy !== "auth0"` if the auth0 middleware checks grow in number.